xmodule/library_sourced_block.py

```diff
@@ -25,7 +39,15 @@
 
 
 @XBlock.wants('library_tools')  # Only needed in studio
-class LibrarySourcedBlock(StudioEditableXBlockMixin, EditableChildrenMixin, XBlock):
+@XBlock.wants('studio_user_permissions')  # Only available in studio
+class LibrarySourcedBlock(
+    StudioEditableXBlockMixin,
+    MakoTemplateBlockBase,
+    XModuleToXBlockMixin,
+    ResourceTemplates,
+    XModuleMixin,
+    EditableChildrenMixin,
```

Comment:
Nit/Q: Going from a React-based editor to using insecure Mako templates and using all these old XModule classes seems like a step backwards... is it because we're re-using a lot of the vertical/container edit code now?

Reply:
(library source block is removed)